model/Service/NavigationContextAwareInterface.php

```diff
@@ -26,5 +26,5 @@
 
 interface NavigationContextAwareInterface
 {
-    public function setNavigationContext(string $direction, string $scope, ?string $ref): void;
+    public function setNavigationContext(?string $direction, ?string $scope, ?string $ref): void;
```

Comment:
This is now a breaking change, but it seems the only place implementing it is the `NavigationContextAwareTrait` trait

https://github.com/search?q=org%3Aoat-sa+setNavigationContext%28&type=code

IMO we do not need to release a breaking change in this case.

Reply:
I agree. Since we are setting to optional parameters that previously were non-optional, I think we can keep it as non-BC if that's ok for you too